test/__snapshots__/step-function.localstack.test.ts.snap

```diff
@@ -1,8 +1,1113 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`binaryOps logic 1`] = `
```

Comment:
I worry about the readability of these state machines

Reply:
Also, cost. Do we have an idea of what % overhead we add in terms of transitions? I see a lot of them, but I assume that's more a measure of the number of branches, not the number of transitions expected in a single execution?

Reply:
What's more important, TS coverage syntax or readability and cost? 

Def thinking about cost and readability, but also doing what we need to make things work in native step functions.

Reply:
Coverage for sure.

Reply:
Yeah. Sfn is hard... I'm refactoring right now in a way that can be a bit smarter. Not leveraging a-normal form when we don't need to, etc. Allows the nodes to optimize themselves.